package.json

```diff
@@ -3,18 +3,20 @@
   "version": "0.0.0-development",
   "description": "Helper module to get AWS polly voices in an idiomatic way",
   "main": "dist/index.js",
+  "types": "src/index.d.ts",
```

Comment:
We should use a file generated inside the `dist` folder, in order to always use the latest version of the types.